_data/simple-icons.json

```
@@ -13049,8 +13049,9 @@
         },
         {
             "title": "Tauri",
-            "hex": "FFC131",
-            "source": "https://github.com/tauri-apps/tauri/blob/093f85dc2b90a6dd0f48d941f6e88daec311250a/app-icon.png",
+            "hex": "24C8D8",
```

```suggestion
            "hex": "070707",
```
Should we be using this instead based on page 5 and 12 here: https://github.com/tauri-apps/tauri-docs/blob/b1cdfa9d7c6d0b17dae60a90266ddced40a7b384/static/img/Brand_Guidelines.pdf

---

There are two colors named with their brand, so I assumed it was the brand color. I took the first one, which is Tauri Blue. Looking at their [website](https://tauri.app/), it looks to be representative of the brand.
We can ask another @simple-icons/maintainers opinion.

---

I agree with Tauri blue being used. It's the first of the two listed in the document, as well as the primary colour on the website.

---

Their intentions seem good to ensure readability in all scenarios for everyone. Also, this is how they use it in a badge on their own GitHub space: https://github.com/tauri-apps/awesome-tauri#guides

---

The goal of the Hex code in Simple Icons is keeping the color that represents the brand, not necessarily the icon color nor the color that looks better on a badge. In this case, as I mentioned, I think we should choose Tauri blue or Tauri Orange.

By looking at the badge from their guides you mentioned, the main color they use is Tauri orange, not necessarily `#070707`.

---

Agreed. Let's get this merged.